openapi/schemas/processes-core/statusInfo.yaml

```diff
@@ -1,42 +1,99 @@
-type: object
-required:
-   - id
-   - status
-   - type
-properties:
-   id:
-      type: string
-   processID:
-      type: string
-   type:
-      type: string
-      example:
-        - process
-        - wps
-        - openeo
-   status:
-      $ref: "statusCode.yaml"
-   message:
-      type: string
-   exception:
-      $ref: "../common-core/exception.yaml"
-   created:
-      type: string
-      format: date-time
-   started:
-      type: string
-      format: date-time
-   finished:
-      type: string
-      format: date-time
-   updated:
-      type: string
-      format: date-time
-   progress:
-      type: integer
-      minimum: 0
-      maximum: 100
-   links:
-      type: array
-      items:
-         $ref: "../common-core/link.yaml"
+allOf:
+  - $ref: descriptionType.yaml
+  - type: object
+    required:
+       - id
+       - type
+       - status
+    properties:
+       id:
+          type: string
+       type:
+          type: string
+          description:
+            An indication of the API that created this job.
```

Comment:
You will need a `|` to do the multiline string and keep the newlines.

Same thing for all following `description` fields.